### ISSUE TITLE
Grid DateFilterCellComponent no weekNumber binding

### ISSUE THREAD

**Issue by dummdidumm** [telerik/kendo-angular#2117]
**Describe the bug**
DateFilterCellComponent has no possibility to set the weekNumber-Option.
https://www.telerik.com/kendo-angular-ui/components/grid/api/DateFilterCellComponent/ 

**Expected behavior**
There should be a weekNumber-Binding that is passed on to the datepicker:
```html
<!-- ..... -->
<kendo-grid-date-filter-cell [column]="column"
                                         [operator]="'eq'"
                                         [filter]="filter"
                                         [weekNumber]="true">
</kendo-grid-date-filter-cell>
```

**Browser**
 - OS independent
 - Browser independent
 - Version: Latest

**Comment by ipeshev** [comment]
Feature available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by danielkaradachki** [comment]
Released in 3.14.0
